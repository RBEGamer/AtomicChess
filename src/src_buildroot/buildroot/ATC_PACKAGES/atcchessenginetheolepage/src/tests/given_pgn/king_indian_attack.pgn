[Event "?"]
[Site "?"]
[Date "????.??.??"]
[Round "?"]
[White "?"]
[Black "?"]
[Result "*"]
[Opening "King's indian attack"]

1. e2-e4 e7-e6 2. d2-d3 d7-d5 3. Nb1-d2 Ng8-f6 4. g2-g3 c7-c5 5. Bf1-g2
Nb8-c6 6. Ng1-f3 Bf8-e7 7. O-O O-O *

