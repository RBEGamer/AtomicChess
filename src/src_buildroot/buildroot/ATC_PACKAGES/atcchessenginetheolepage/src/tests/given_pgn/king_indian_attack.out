[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 3
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 0
[kingside castling]	color: 0
[move]	board::PieceType: 0
[kingside castling]	color: 1
